src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -394,27 +403,22 @@ public void insertIntoDatasetTable(Dataset dataset,
          */
         BigQueryProject bigQueryProject = bigQueryProjectForDataset(dataset);
         String projectId = bigQueryProject.getProjectId();
-        StringBuilder sql = new StringBuilder();
-        sql.append("INSERT ")
-            .append("`")
-            .append(projectId)
-            .append(".")
-            .append(prefixName(dataset.getName()))
-            .append(".")
-            .append(targetTable.getName())
-            .append("` (");
-        buildColumnList(sql, targetTable, true);
-        sql.append(") SELECT ");
-        buildColumnList(sql, targetTable, true);
-        sql.append(" FROM `")
-            .append(projectId)
-            .append(".")
-            .append(prefixName(dataset.getName()))
-            .append(".")
-            .append(stagingTableName)
-            .append("`");
-
-        bigQueryProject.query(sql.toString());
+        StringBuilder sqlTemplate = new StringBuilder();
+
+        sqlTemplate.append("INSERT `{targetTableName}` (");
+        buildColumnList(sqlTemplate, targetTable, true);
+        sqlTemplate.append(") SELECT ");
+        buildColumnList(sqlTemplate, targetTable, true);
+        sqlTemplate.append("From `{stagingTableName}`");
```

Comment:
IMO, mixing `StringBuilder` with the interpolator makes it more difficult to tell what's going on here. I'd stick with one or the other.

Reply:
I would slightly prefer one or the other; however, have no hard feeling about this. The reason for doing so was some of the sqlTemplates have to be broken up to either handle if else logic or to be passed to another function the appends to the `StringBuilder`. @thathert and @ahaessly any thoughts?